packages/nextjs/tailwind.config.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
 
           "--btn-text-case": "none",
 
-          "--rounded-btn": "9999rem",
+          "--rounded-btn": "8px",
```

Comment:
We can also completely remove this from here since by-default daisy sets "--rounded-btn : 0.5rem" i.e `8px`

![Screenshot 2023-07-30 at 8 11 00 PM](https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/assets/80153681/bb8f4e55-6f7a-4afb-96c5-e202cb3abd1e)

---

Btw I would suggest updating it to `"--rounded-btn": "0.75rem"` i.e `12px` because in Figma design they are following `12px` border-radius for all buttons : 

![Screenshot 2023-07-30 at 8 20 47 PM](https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/assets/80153681/9ee18d54-0818-40ae-bd36-1168e38306e0)

---

Also after updating, we should probably also probably remove `rounded-xl` from here and add `btn` : 
https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/blob/22025d964e68ee74a81fd2af669366b4b7666d2f/packages/nextjs/components/op/btn/CustomProjectButton.tsx#L13


Because adding `btn` will not only add correct rounded border but it also comes with some by default goodies like animation's : 

https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/assets/80153681/0f4efba0-5a58-4378-85e0-b0bf486e6586

You can also set custom hover / other styles for `btn` matching to figma checkout Pt.3 ->  https://v2.daisyui.com/docs/customize

Sorry for the long review 😅, but since this was fast prototyping its ok to have some stuff hardcoded  and we can clean up soon in later PR's 🙌

Reply:
Thanks for the feedback. I made these adjustments in the latest commit.